ocs_ci/ocs/cluster.py

```diff
@@ -903,6 +904,84 @@ def get_osd_utilization():
     return osd_filled
 
 
+def get_ceph_df_detail():
+    """
+    Get ceph osd df detail
+    Returns:
+         dict returned from cmd
```

Comment:
dict: The returned output from 'ceph df detail' command

Reply:
Done